modules/core/test/com/haulmont/addon/ldap/core/MatchingRuleTest.java

```diff
@@ -48,783 +49,796 @@
 
 public class MatchingRuleTest {
 
-    private static final String CUSTOM_RULE_ID = "com.haulmont.addon.ldap.core.custom.TestCustomLdapRule";
```

Comment:
Did you change the code formatting here? I can't see actual changes in this diff

Reply:
Yes, I think I changed it but not specially. May be it happens because I use tab format - spaces, not tabs (standart Intellij tabs settings)